### ISSUE TITLE
5mL syringe, not 6mL

### ISSUE THREAD

**Issue by joeborrello** [megmrod/Slic3r#1]

**Comment by megmrod** [comment]
Because those syringes mark up to 6 mL, included 6 ml in the presets. The 5mL preset is now included in the repo.

**Comment by joeborrello** [comment]
The McMaster syringes go to 6mL?

**Comment by megmrod** [comment]
Yes, but after digging through their website, it looks like it's as extra because the 6mL marking is dotted, and it's after the solid 5mL/cc marking. They differ only by 0.20 mm in ID.

**Comment by joeborrello** [comment]
Ah okay, that's a weird quirk. It just occurred to me that we should also have a text or CSV file on this repo with the McMaster part numbers for each of the syringes and syringe tips

**Comment by megmrod** [comment]
Ok, I can work on that.
